Sources/RInAppMessaging/Utilities/AnalyticsTracker.swift

```diff
@@ -0,0 +1,16 @@
+import class UIKit.UIDevice
+
+#if SWIFT_PACKAGE
+import class RSDKUtilsMain.AnalyticsBroadcaster
+#else
+import class RSDKUtils.AnalyticsBroadcaster
+#endif
+
+internal struct AnalyticsTracker {
+    static func sendEventName(_ name: Constants.RAnalytics, dataObject: [String: Any]? = nil) {
```

Comment:
Can't you just put `[:]` as default value instead of `nil`?
Then you could also remove line 11

Reply:
Done